### PR TITLE
XA Two phase commit support

### PR DESCRIPTION
**Pull request by endurox-dev**
Fixed issues with previous request.

### CODE REVIEW

File: oci8.go [hunk 1]
```diff
@@ -216,6 +216,28 @@ WrapOCIEnvCreate(ub4 mode, size_t extra) {
   return vvv;
 }
 
+static ret2ptr
+WrapxaoEnv() {
+  ret2ptr vvv = {NULL, NULL, 0};
+
+#ifdef OCI8_ENABLE_XA
+  vvv.ptr = (dvoid *)xaoEnv(NULL);
```
Comment (mattn) [inline oci8.go]: You can separate files oci8_xa.go which include this part and build-constraint tag like below.

```go
// +build xa

package oci8

/*
...
static ret2ptr
WrapxaoEnv() {
  ret2ptr vvv = {NULL, NULL, 0};
  vvv.ptr = (dvoid *)xaoEnv(NULL);
  return vvv;
}

static ret2ptr
WrapxaoSvcCtx() {
  ret2ptr vvv = {NULL, NULL, 0};
  vvv.ptr = (dvoid *)xaoSvcCtx(NULL);
  return vvv;
}
```

Comment (endurox-dev) [reply oci8.go]: OK will fix.

**Comment by MichaelS11** [comment]
Close this?

**Comment by endurox-dev** [comment]
we can close it, as I will open new one, once change will be developed.

**Pull request closed without merging**